force-app/main/default/classes/SOQL.cls

```diff
@@ -117,7 +162,13 @@ public with sharing class SOQL {
      * @return A list of Case records.
      */
     public static List<Case> getFilterCaseOrigins() {
-        return null; // Replace null with the variable you used to store the result
+        List<Case> cases = [
+            SELECT Id, Origin, Status, AccountId
+            FROM Case
+            WHERE Origin NOT IN ('Web', 'Email', '') 
+            AND Origin != null
```

Comment:
In SOQL null and empty string evaluate to the same, so you shouldnt need the extra != NULL, but it is not hurting anything.